.github/workflows/PR_summary.yml

```
@@ -104,23 +104,31 @@ jobs:
       run: |
         cd pr-branch
         touch moved_without_deprecation.txt
+        touch extraneous_deprecated_module.txt
         git checkout ${{ github.base_ref }}
         while IFS= read -r file
         do
           if grep ^deprecated_module "${file}" ; then
-            printf 'info: removed file %s contains a deprecation\n' "${file}"
+            # shellcheck disable=SC2016
+            printf '\n⚠️ **warning**: removed file `%s` contains a module deprecation: please add this in a follow-up PR instead\n' "${file}" |
+              tee -a extraneous_deprecated_module.txt
           else
             # shellcheck disable=SC2016
-            printf '\n⚠️ **warning**: file `%s` was removed without a module deprecation\n' "${file}" |
+            printf '\nnote: file `%s` was removed.\nPlease create a follow-up pull request adding a module deprecation. Thanks!\n' "${file}" |
               tee -a moved_without_deprecation.txt
           fi
         done < removed_files.txt
         IFS=$'\n'
         while IFS= read -r file
         do
-          # shellcheck disable=SC2016
-          printf '\n⚠️ **warning**: file %s without a module deprecation\n' "${file}" |
-            tee -a moved_without_deprecation.txt
+          if grep ^deprecated_module "${file}" ; then
+            # shellcheck disable=SC2016
+            printf '\n⚠️ **warning**: renamed file `%s` contains a module deprecation: please add this in a follow-up PR instead\n' "${file}" |
+              tee -a extraneous_deprecated_module.txt
+          else
+            printf '\nnote: file %s without a module deprecation\nPlease create a follow-up pull request adding one. Thanks!\n' "${file}" |
```

```suggestion
            # shellcheck disable=SC2016
            printf '\nnote: file `%s` without a module deprecation\nPlease create a follow-up pull request adding one. Thanks!\n' "${file}" |
```
This is just to maintain uniformity of backticks: feel free to ignore!